src/mango/src/mango_idx.hrl

```diff
@@ -13,9 +13,11 @@
 -record(idx, {
     dbname,
     ddoc,
+    ddoc_id,
```

Comment:
I'm still confused on whether this change is a field rename or we're actually adding a second field.

Reply:
I don't remember ever seeing a place where the design document was stored in that field. I specifically checked `mango_idx:from_ddoc` which doesn't do it. I don't remember other places where that might have been set separately.

Reply:
I was adding the ddoc in the mango_idx:list but you right that isn't intuitive. I've moved it into the from_ddoc function